cmd/tempo/app/app.go

```diff
@@ -85,12 +85,12 @@ func (c *Config) RegisterFlagsAndApplyDefaults(prefix string, f *flag.FlagSet) {
 
 // CheckConfig checks if config values are suspect.
 func (c *Config) CheckConfig() {
-	if c.Ingester.CompleteBlockTimeout < c.StorageConfig.Trace.MaintenanceCycle {
+	if c.Ingester.CompleteBlockTimeout < c.StorageConfig.Trace.BlocklistPoll {
 		level.Warn(util.Logger).Log("msg", "3ingester.CompleteBlockTimeout < storage.trace.maintenance-cycle",
```

Comment:
there's a weird 3 here.  let's clean that up :)

"3ingester.CompleteBlockTimeout"

Reply:
Good catch @joe-elliott! Probably some leftover from testing :stuck_out_tongue_closed_eyes: